support/build/Makefile

```diff
@@ -0,0 +1,101 @@
+#==============================================================================
+# Copyright (C) 2018 Stephen F Norledge & Alces Software Ltd.
+#
+# This file is part of Alces Cloudware.
+#
+# Some rights reserved, see LICENSE.
+#==============================================================================
+# Image config
+PLATFORM=azure # default to azure
+IMAGE_TYPE=alces-cloudware-base
+IMAGE_VERSION=2018.2.1
+IMAGE_NAME=$(IMAGE_TYPE)-$(IMAGE_VERSION)-$(PLATFORM)
+
+# Libvirt/Oz config
+KICKSTART=$(IMAGE_TYPE)-$(PLATFORM).ks
+KICKSTART_RENDERED=/tmp/$(IMAGE_NAME).ks
+TDL=centos7.tdl
+TDL_RENDERED=/tmp/$(IMAGE_NAME).tdl
+OZ_CFG=oz.cfg
+VM_DIR=/mnt/resource
+QEMU_IMG_BIN=$(VM_DIR)/qemu-img
+XML=domain.xml
+XML_RENDERED=$(IMAGE_NAME).xml
+
+# AWS config
+AWS_BUCKET=alces-cloudware
+AWS_BUCKET_DIR=images
+AWS_REGION=eu-west-1
+
+# Azure config
+AZURE_STORAGE_ACCOUNT=alcescloudware
+AZURE_STORAGE_CONTAINER=images
+AZURE_RESOURCE_GROUP=alces-cloudware
+AZURE_IMAGE_URL="https://$(AZURE_STORAGE_ACCOUNT).blob.core.windows.net/$(AZURE_STORAGE_CONTAINER)/$(IMAGE_NAME).vhd"
+
+image: setup build prepare upload
+
+setup:
+	[ -d $(VM_DIR)/converted ] || mkdir -p $(VM_DIR)/converted
+	[ -d $(VM_DIR)/tmp ] || mkdir -p $(VM_DIR)/tmp
```

Comment:
Do you need the test here? Doesn't `mkdir -p` handle the case where the dir already exists fine?